scapy/plist.py

```diff
@@ -299,10 +306,15 @@ def conversations(self, getsrcdst=None,**kargs):
             except:
                 #XXX warning()
```

Comment:
I have updated the comment to explain why it might not be ideal to add a warning here. What do you think?